### DISCUSSION TITLE
Self-Introduction: Welcome to SwissGerman_Dictionary Discussions!

### DISCUSSION THREAD

**Discussion by esthicodes** [Estheryu991/SwissGerman_Dictionary#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by timonreichert** [comment]
Ich bin de Timon, vo de Bade, i de Schwiz und studier Mathi im 3. Semester vom Master. Mis Lieblingsfach isch Algebraischi Geometrie.
Min Lieblingstee isch Schwarztee. Mis Lieblingsesse isch Dampfnudle.
Dampfnudle sind e traditionelli Mählspiis vo de süddütsche Chuchi.

**Comment by dennis54321** [comment]
Mis Lieblingsesse isch Shepherd‘s pie
